packages/fluentui/react-northstar/test/specs/components/Toolbar/Toolbar-test.tsx

```diff
@@ -62,4 +62,24 @@ describe('Toolbar', () => {
       expect(getRenderedAttribute(italicToolbarButton, 'aria-pressed', '')).toBe('false');
     });
   });
+
+  describe('Overflow Toolbar', () => {
+    it('Should override popper properties', () => {
+      const wrapper = mountWithProvider(
+        <Toolbar
+          aria-label="Toolbar overflow menu"
+          items={[]}
+          overflow
+          overflowItem={{ title: 'More', menu: { popper: { offset: [0, 4], position: 'below' } } }}
+        />,
+      );
+
+      const overflowItem = wrapper.find('ToolbarItem').last();
+
+      expect(overflowItem.prop('menu')).toMatchObject({
```

Comment:
nit: do you want to use `toMatchObject` here ? it works off partial matching. So if you want an exact match then `.equals` might be better

Reply:
@layershifter suggestion looks cleaner, feel free to ignore this